app/components/navbar.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react'
+import type {ReactNode} from 'react'
+import {SunIcon} from './icons/sun-icon'
+import {MoonIcon} from './icons/moon-icon'
+import {MenuIcon} from './icons/menu-icon'
+
+interface NavLinkProps {
+  href: string
+  active?: boolean
+  children?: ReactNode
+}
+
+function NavLink({href, children}: NavLinkProps) {
+  return (
+    <li>
+      <a
```

Comment:
I think we should replace this `<a>` with `Link` from `react-router-dom`, but storybook stopped working then. We should add a decorator to `preview.js` to wrap all stories in `react-router`.

Reply:
Cool, good to know. I'll take care of this so you can keep pushing forward on what you're working on.

Reply:
Added a note to the project. Let's do this when we're connecting things to Remix, and know where the links should point to.

https://github.com/kentcdodds/remix-kentcdodds/projects/1#card-63944993